.github/workflows/hotfix.yml

```diff
@@ -0,0 +1,40 @@
+name: "Hotfix"
+on:
+  pull_request:
+    branches:
+      - master
+    types: [closed]
+
+
+jobs:
+  build:
+    if: github.event.action == 'closed' && github.event.pull_request.merged == true && contains(github.event.pull_request.head.ref, 'hotfix/')
+
+    runs-on: ubuntu-16.04
+    container: rappdw/docker-java-python
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Install dependencies
+      run: make ci-install
+
+    - name: Build package
+      run: make package
+
+    - name: Get version
+      run: echo ::set-env name=version::$(grep __version__ setup.py | head -1 | cut -d \" -f2 | cut -d \' -f2)
+
+    - name: Create release
+      uses: actions/create-release@v1
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        tag_name: ${{ env.version }}
+        release_name: Release ${{ env.version }}
+
+    - name: Publish release to pypi.org
+      env:
+        PYPI_USERNAME: ${{ secrets.PYPI_USERNAME }}
+        PYPI_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
+      run:  PYTHONPATH=./pip/deps python -m twine upload -u $PYPI_USERNAME -p $PYPI_PASSWORD --verbose dist/*
```

Comment:
All of these steps are exactly the same as the Publsh workflow. For what I'm seeing for a release and for a hotfix they should run the same steps. So maybe we need just **one workflow** that checks if release or hotfix conditions are satisfied and trigger these steps. The problem in just copy and paste this steps is:  if change the logic on how to publish we will always need to remember to change both files.

Reply:
I agree about the maintenance of two files, so what do you think one workflow for publishing but this workflow has two jobs with different conditions?